src/app.js

```diff
@@ -1 +1,42 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { generateRandomNumber } = require('./generateRandomNumber');
+const { isValid } = require('./isInputValid');
+const { terminal } = require('./terminal');
+const { calculateBullsAndCows } = require('./calculateCowsAndBulls');
+
+const generetadNum = generateRandomNumber();
+
+const checkUserAnswer = (answer) => {
+  if (!isValid) {
+    console.log('Insert a number of 4 different digits');
```

Comment:
For all these strings, you can use JSON to store all your text data. Your code will be cleaner and it better for future uses. Because if you want to add some new data or reuse your text message, your data will be in one place, and you can easily access it.